tests/common/mention_free_text_test.py

```diff
@@ -0,0 +1,28 @@
+from unittest.mock import MagicMock
+
+import pytest
+
+from common.mention_free_text import MentionFreeText, NotMentionFreeException
+
+
+@pytest.mark.asyncio
+async def test_convert_returns_text_if_clean(ctx):
+    text = "just because you have a meme command doesn't mean you're dank"
+    msg = MagicMock(content=text, mentions=[], role_mentions=[])
+
+    assert (await MentionFreeText.convert(MagicMock(spec=ctx, message=msg), text)) == text
+
+
+@pytest.mark.asyncio
+async def test_convert_raises_error_if_not_clean(ctx):
```

Comment:
I need to sit through and understand the whole codebase before it gets too large, but thank you for remembering tests :)